cornac/models/recvae/recom_recvae.py

```diff
@@ -0,0 +1,280 @@
+# Copyright 2018 The Cornac Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+import numpy as np
+
+from ..recommender import Recommender
+from ...exception import ScoreException
+from copy import deepcopy
+
+
+from tqdm.auto import trange
```

Comment:
maybe move this import up under the `import numpy`?

Reply:
Yes. I've moved it.